staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -583,7 +583,9 @@ func (p *patcher) patchResource(ctx context.Context, scope *RequestScope) (runti
 	}
 
 	wasCreated := false
-	p.updatedObjectInfo = rest.DefaultUpdatedObjectInfo(nil, p.applyPatch, p.applyAdmission, dedupOwnerReferencesTransformer)
+	p.updatedObjectInfo = rest.DefaultUpdatedObjectInfo(nil, p.applyPatch, p.applyAdmission, dedupOwnerReferencesTransformer,
+		fieldmanager.DropEqualTransformer) // Ignore changes that only affect managed fields timestamps.
```

Comment:
That means the equality check happens after we do the managedFields update, right?
Does that mean semantically equal changes would still update the managedFields as long as they go with other "real" changes?
But if they come on their own they get dropped.
Is that an acceptable inconsistency to fix the problem?

So if you change a value that means the same while making a relevant change. You would still own it (which makes sense in most cases I guess), but if you only make that change nothing would happen, right?